bls/src/main/java/org/dashj/bls/Uint8Vector.java

```diff
@@ -0,0 +1,144 @@
+/* ----------------------------------------------------------------------------
+ * This file was automatically generated by SWIG (http://www.swig.org).
+ * Version 4.0.2
+ *
+ * Do not make changes to this file unless you know what you are doing--modify
+ * the SWIG interface file instead.
+ * ----------------------------------------------------------------------------- */
+
+package org.dashj.bls;
+
+public class Uint8Vector extends java.util.AbstractList<Short> implements java.util.RandomAccess {
+  private transient long swigCPtr;
+  protected transient boolean swigCMemOwn;
+
+  protected Uint8Vector(long cPtr, boolean cMemoryOwn) {
+    swigCMemOwn = cMemoryOwn;
+    swigCPtr = cPtr;
+  }
```

Comment:
This class wraps `vector<uint8>` which in Java is `byte[]`.  SWIG uses `Short` instead of `byte`.